rustworkx/visualization/graphviz.pyi

```diff
@@ -10,17 +10,68 @@ import typing
 from rustworkx.rustworkx import PyGraph, PyDiGraph
 
 if typing.TYPE_CHECKING:
-    from PIL.Image import Image  # type: ignore
+    from typing_extensions import TypeAlias
+    from PIL.Image import Image
+
+    _Method: TypeAlias = typing.Literal["twopi", "neato", "circo", "fdp", "sfdp", "dot"]
+    _ImageType: TypeAlias = typing.Literal[
```

Comment:
Per #830, this still can fail at runtime so again I'd rather just make the type as a string

Reply:
I inlined the Literal, it shouldn't fail at runtime anymore